### PR TITLE
tests/plugins/lightline: fix test

### PR DESCRIPTION
**Pull request by khaneliman**
Resolves https://github.com/nix-community/nixvim/issues/1934 which is also blocking https://github.com/nix-community/nixvim/pull/2076

### CODE REVIEW

File: tests/test-sources/plugins/statuslines/lightline.nix [hunk 1]
```diff
@@ -113,7 +113,7 @@
       enable = true;
 
       settings = {
-        colorscheme = "gruvbox";
+        colorscheme = "one";
```
Comment (MattSturgeon) [inline tests/test-sources/plugins/statuslines/lightline.nix:116]: Also need to update the settingsExample, as per 4d012f8eb23f177a1b8a326ce1148a8837c7b55d

**Comment by MattSturgeon** [comment]
Can you add a note (and a link?) to your commit message?

**Review by MattSturgeon — APPROVED** [review]
Awesome, thanks for the quick fix.

**Review by traxys — APPROVED** [review]

**Comment by khaneliman** [comment]
@mergifyio queue

**Pull request merged**